codegen/writers/endpoint.go

```diff
@@ -29,19 +32,80 @@ type (
 	}
 )
 
+type endpointWriter struct {
+	sections   []*codegen.Section
+	outputPath string
+}
+
+func (e endpointWriter) Sections() []*codegen.Section {
+	return e.sections
+}
+
+func (e endpointWriter) OutputPath() string {
+	return e.outputPath
+}
+
 // EndpointsWriter returns the codegen.FileWriter for the endpoints of the given
 // service.
 func EndpointsWriter(api *design.APIExpr, service *design.ServiceExpr) codegen.FileWriter {
-	return nil
+	return codegen.FileWriter(endpointWriter{
```

Comment:
It was not needed. I fixed it. https://github.com/goadesign/goa/pull/1090/commits/675ab2cd0e6315d4ff8f935050c17d5647f31607